src/app/content/state/click-count/click-count.reducer.ts

```diff
@@ -0,0 +1,12 @@
+import { createReducer, on } from '@ngrx/store';
+import * as ClickCountActions from './click-count.actions';
+import {initialState} from "./click-count.state";
+
+export const clickCountReducer = createReducer(
+  initialState,
+  on(ClickCountActions.loadClickCount, (state, { count, lastUpdated }) => ({ count, lastUpdated })),
+  on(ClickCountActions.incrementClickCount, state => {
```

Comment:
ah you have it already .. beware of putting increment logic in two places .. now you are adding some stuff here and some stuff in the effects, where also mixing up something in the localStorage

Reply:
this I was also thinking and being concerned about, I will try to use effects mostly for it